qa/tasks/cephadm.py

```diff
@@ -477,7 +475,17 @@ def ceph_bootstrap(ctx, config):
                 log.exception(f'Failed to stop "{role}"')
                 raise
 
+        # stop the catch-all, too, so that we also stop daemons/roles we
+        # don't know about  Note that doesn't block until they are all stopped, so
+        # wait a moment
+        log.info(f'Stopping ceph-{fsid}.target...')
+        ctx.cluster.run(args=['sudo', 'systemctl', 'stop', f'ceph-{fsid}.target'])
+
+        log.info('Waiting a few moments for containers to stop...')
+        time.sleep(10)
```

Comment:
sleeps like this seem prone to racing with e.g. log gathering, I suppose we can see how much it comes up in practice though